maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -808,6 +808,18 @@
     @Parameter
     private Map<String, String> jdkToolchain;
 
+    /**
+     * How often output is forced to be flushed.
+     * Useful when there is no output for N ms but some data are buffered.
+     * It will trigger a flush each configured interval to ensure
+     * data don't stay blocked in a buffer.
+     * Setting it to 0 disable that feature.
+     *
+     * @since 3.0.0-M6
+     */
+    @Parameter( defaultValue = "0" )
```

Comment:
why about setting the default to 500ms ?
otherwise people will add some value to every pom.xml
if we have a sensible default then we will save users requests on the ML and pom pollution

Reply:
Hope is that it gets removed in 3.0.0 and does not change default behavior so used 0 (disable) as default.

Reply:
I see